<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tumasulo算法演示平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Instructions:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="insTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>56</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>15</number>
          </attribute>
          <column>
           <property name="text">
            <string>Index</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Op</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dest</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Srci</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Srck</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>State</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Resvervation Stations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="rsTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rowCount">
           <number>5</number>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>15</number>
          </attribute>
          <row>
           <property name="text">
            <string>Add1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Add1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Add3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Mult1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Mult2</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Busy</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Op</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vj</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vj</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Vk</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Qk</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Load/Store Queue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="lsQueueTable">
          <property name="rowCount">
           <number>1</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>15</number>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>3</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>4</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>5</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>6</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Load Buffer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="loadBufferTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>15</number>
          </attribute>
          <row>
           <property name="text">
            <string>Load1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Load2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Load3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Busy</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Qi</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Store Buffer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="storeBufferTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>15</number>
          </attribute>
          <row>
           <property name="text">
            <string>Store1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Store2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Store3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Busy</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cache</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Registers</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="registerTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <row>
         <property name="text">
          <string>Qi</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Value</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>F0</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F5</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F6</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F7</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F8</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F9</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F10</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>label_4</zorder>
   <zorder>loadBufferTable</zorder>
   <zorder>storeBufferTable</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
